java/test/TestH5.java

```diff
@@ -34,6 +34,8 @@
 import hdf.hdf5lib.exceptions.HDF5Exception;
 import hdf.hdf5lib.exceptions.HDF5LibraryException;
 
+import test.H5TestUtils;
```

Comment:
Why is this imported into in every java file?
imports should only be in the file that uses that class.